### ISSUE TITLE
Mazumder, Crandall, Pond - Planned data release for Feb 22nd (V1.44-1.45)

### ISSUE THREAD

**Issue by steph-sing** [FDA-ARGOS/data.argosdb#211]

**Comment by steph-sing** [comment]
Push is rescheduled for Wednesday Feb 28th due to changes in datasets.

**Comment by steph-sing** [comment]
Primary and Secondary Data Release Notes Draft: 

- [ ] @kee007ney Please push to all Argos Instances 
- [ ] Once pushed, @steph-sing will notify @HadleyKing and @Jgergely11 

Input by: Steph
Description: 6 tables were combined to result in 3 larger datasets (BCOs and recipes updated). 1 new fasta record with 2 fasta inside (new BCO and recipe). 5 datasets updated. 
Location: Server: Dev 
(Output: /reviewed folder in Dev) 

New Tables (Primary Release):
ARGOS_000025 - ngsQC_Crandall.tsv absorbed into ARGOS_000019 - ngsQC_HL.tsv. 
New BCO and File name = ARGOS_000019 - ngsQC_HIVE.tsv
assemblyQC-Crandall.tsv absorbed into ARGOS_000012 - assemblyQC_HL.tsv. 
New BCO and File name = ARGOS_000012 - assemblyQC_HIVE.tsv
ARGOS_000028 - biosampleMeta-Crandall.tsv absorbed into ARGOS_000020 - biosampleMeta_HL.tsv. 
New BCO and File name = ARGOS_000020 - biosampleMeta_HIVE.tsv
ARG_000010 - reference-guided_genome_assemblies_HIVE-Hexagon.fasta

Updated Tables (Primary Data Release): 
ARGOS_000011 - siteQC_HIVE.tsv

Updated Tables (Secondary Data Release): 
ARGOS_000038 - assemblyQC_NCBI
ARGOS_000009 - ngsQC_NCBI
ARGOS_000018 - ngs_id_list.tsv
ARGOS_000010 - biosampleMeta_ncbi.tsv (previously SRA_biosample.tsv; table was updated but BCO still needs to be updates) 


Remaining Tables in /reviewed: 
ARG_000001 - NC_045512_SARS-CoV-2_Wuhan.fasta
ARGOS_000014 - sars-cov-2_lineage_mutations.tsv

Cards to Archive in the DB (Will Request Space from Robel - still currently in /review - no action): 
ARGOS_000013 - HIV1_UP000002241_proteome_metadata.csv 
ARGOS_000029 - UP000180448_33727_DNA.fasta
ARGOS_000030 - UP000180448_33727.fasta
ARGOS_000031 - uniprot-proteome_UP000180448.csv
ARGOS_000032 - UP000001014_99287_DNA.fasta
ARGOS_000033 - UP000001014_99287.fasta
ARGOS_000034 - uniprot-proteome_UP000001014.csv
ARGOS_000001 - UP000009255_211044_DNA.fasta
ARGOS_000002 - UP000009255_211044.fasta
ARGOS_000003UP000009255_proteome_genome_metadata.csv
ARGOS_000004 - UP000464024_2697049_DNA.fasta
ARGOS_000005 - UP000464024_2697049.fasta
ARGOS_000006 - uniprot-proteome_UP000464024.csv
ARGOS_000008 - PRJNA231221_AssemblyUpdated.tsv

**Comment by steph-sing** [comment]
@rykahsay 
It seemed like all instances went blank when trying to push V1.43, so we had to roll things back to V1.42. I've attached a screenshot and draft release notes to assist in our troubleshooting, since neither Jonathon or I have experienced this issue before and were unable to resolve it ourselves. Can yo please assist? (Draft Release Notes above - happy to explain any changes)


![Image](https://user-images.githubusercontent.com/70154794/222481861-dd508f6e-c10a-4d03-8173-3bafd1e8325a.png)

**Comment by rykahsay** [comment]
The script "object-maker/make-siteqcdb.py" is failing and I have modified it so that it outputs the right error message (see screenshot below).  The problem is that it expects the field "genomic_coordinates_start" and what you have in the siteQC_HIVE.tsv and core_property_list.tsv files is "genomic_coordinate_start". To fix this, you need to do the following changes in the files siteQC_HIVE.tsv and core_property_list.tsv.

From "genomic_coordinate_start" to "genomic_coordinates_start"




<img width="1062" alt="image" src="https://user-images.githubusercontent.com/20584876/222501814-15839001-50c3-4475-84f5-e61c74106ef8.png">

**Comment by steph-sing** [comment]
@rykahsay I am not sure why this field keeps being changed back or why it wasn't caught in our pre-release QC scripts for the data dictionary. I will address this, thank you. 

Otherwise I am seeing another issue, outlined below: 
it looks like even with the roll back to V1.42, I am seeing the changes I was intending to make with V1.43. It seems like the previous version of 1.42 was erased and replaced with V1.43? Do you think this will automatically be fixed when we push to V1.43?

Example: in V1.43 I absorbed Crandall's ngsQC table into ours, and changed the name of the file to ngsQC-HIVE (and Crandall's old ngsQC table does not exist in the DB anymore). I see this also reflected in the Release History for V1.41-V1.42
![image](https://user-images.githubusercontent.com/70154794/222511388-fe54e36d-a77c-4e4f-8601-afcf4063a40d.png)
![image](https://user-images.githubusercontent.com/70154794/222511913-74cd5fa0-1830-408e-b131-36f57f10ea6a.png)

**Comment by rykahsay** [comment]
Looks like Jonathon created 1.42 yesterday and nobody touched that release after that. Can you focus in pushing 1.43 correctly and not worry about what happened to 1.42?


<img width="1119" alt="image" src="https://user-images.githubusercontent.com/20584876/222513959-0d084adf-f14d-403d-b009-6bb49891dd61.png">

**Comment by rykahsay** [comment]
@kee007ney --> since you are the one creating the push, can you please try to make 1.43 now? Or you can do a new one (1.44)?

Thanks

**Comment by kee007ney** [comment]
Done. 1.44 pushed to tst/beta/prd.

**Comment by HadleyKing** [comment]
@rykahsay and @steph-sing 
The field changes with `genomic_coordinates_start` and `genomic_coordinate_start` was due to me. When I received the data dictionary the last few times it was throwing errors and I changed it. I made the changes to `siteQC_HIVE.tsv` and `core_property_list.tsv` based on the data dictionary assuming that that was the curated "source of truth". 

I thought that I had reported those changes in a way that would let everyone know what happened in the release documentation. 

We will fix that breakdown in communication for this release
